app/stac_api/admin.py

```diff
@@ -67,8 +82,16 @@ class ItemAdmin(admin.ModelAdmin):
                     'properties_datetime',
                     'properties_start_datetime',
                     'properties_end_datetime',
-                    'properties_title'
+                    'properties_title',
+                    'properties_eo_gsd'
                 )
             }
         ),
     )
+    # customisation of the geometry field
+    map_template = finders.find('admin/ol_swisstopo.html')  # custom swisstopo
+    if os.environ['APP_ENV'] == 'local':
```

Comment:
`APP_ENV` is part of the django settings and should be read from it and not directly from os.environ